src/Pact/Native.hs

```diff
@@ -286,11 +286,10 @@ namespaceDef = setTopLevelOnly $ defRNative "namespace" namespace
 
 sortDef :: NativeDef
 sortDef = defRNative "sort" sort'
-  (funType (TyList a) [("values",TyList a)] <>
-   funType (TyList (tTyObject (mkSchemaVar "o"))) [("fields",TyList tTyString),("values",TyList (tTyObject (mkSchemaVar "o")))])
-  "Sort a homogeneous list of primitive VALUES, or objects using supplied FIELDS list. \
-  \`(sort [3 1 2])` `(sort ['age] [{'name: \"Lin\",'age: 30} {'name: \"Val\",'age: 25}])`"
-
+     (funType (TyList a) [("values",TyList a)] <>
+      funType (TyList (tTyObject (mkSchemaVar "o"))) [("fields",TyList tTyString),("values",TyList (tTyObject (mkSchemaVar "o")))])
+     "Sort a homogeneous list of primitive VALUES, or objects using supplied FIELDS list. \
+     \`(sort [3 1 2])` `(sort ['age] [{'name: \"Lin\",'age: 30} {'name: \"Val\",'age: 25}])`"
```

Comment:
Looks like formatting is really inconsistent in `Native.hs`, and not just here. Can we settle on 4 space indent with 2 space where like we do in Chainweb?

Reply:
I don't have a particular opinion about formatting, and was just trying to follow along. Perhaps @slpopejoy would be the right person to ask.

Reply:
I went through looking to see if this was a 1-off, and it looks like it's very inconsistent throughout. It probably doesn't need to happen in this PR, but it is slightly annoying.

Reply:
My comment is different: please undo changes to this file as they are whitespace-only.

Reply:
Ah, oops! I once had real changes to this file, but they ended up being irrelevant after a merge and I didn't notice. Will do.